Dockerfile

```diff
@@ -83,14 +83,17 @@ RUN apt-add-repository ppa:brightbox/ruby-ng \
 
 ### PYTHON
 
-# Install Python 2.7 and 3.9 with pyenv. Using pyenv lets us support multiple Pythons
+# Install Python 3.10 with pyenv.
 ENV PYENV_ROOT=/usr/local/.pyenv \
   PATH="/usr/local/.pyenv/bin:$PATH"
 RUN mkdir -p "$PYENV_ROOT" && chown dependabot:dependabot "$PYENV_ROOT"
 USER dependabot
-RUN git clone https://github.com/pyenv/pyenv.git --branch v2.2.2 --single-branch --depth=1 /usr/local/.pyenv \
-  && pyenv install 3.10.0 \
-  && pyenv global 3.10.0 \
+# FIXME: Use tagged release when there is release with Python 3.10.2 support
+# RUN git clone https://github.com/pyenv/pyenv.git --branch v2.2.3 --single-branch --depth=1 /usr/local/.pyenv \
+RUN git clone https://github.com/pyenv/pyenv.git /usr/local/.pyenv \
+  && cd /usr/local/.pyenv && git checkout a55dd92436863cae0e2af1c1e051ee5f8853c005 && cd - \
```

Comment:
As noted in your comment, I think it's best if we use a tagged release of `pyenv`. Rather than checking out an unreleased commit, let's hold off until `v2.2.4` is released.

Reply:
For previous pyenv releases, it took some time to create a new release with the latest Python. I wouldn't prefer to wait them if that's up to me, but I created a issue on their side to hopefully speed up the process https://github.com/pyenv/pyenv/issues/2224

Reply:
Pyenv released 2.2.4 and i updated the PR.